include/roboteam_utils/Polygon.h

```diff
@@ -37,7 +37,7 @@ namespace rtt {
          * 
          * @param vertices Vertices that will literally be copied to this->vertices
          */
-        Polygon(const std::vector<Vector2> &vertices);
+        Polygon(std::vector<Vector2> vertices);
```

Comment:
This doesn't seem to add up to the constructor in Polygon.c, which expects "_vertices" instead of "vertices"

https://github.com/RoboTeamTwente/roboteam_utils/blob/502da26ae21b53d93bfa365b9dca6b0e09c5fa4b/src/Polygon.cpp#L20

Reply:
It's not a bug it is inconsistent so I'll fix it.

Reply:
do you std::move() this after?